### DISCUSSION TITLE
비회원 주문 관련 제안

### DISCUSSION THREAD

**Discussion by Jummi10** [cau-swe/checkpoint-1#8] [General]
주문과 관련된 Requirements와 Use Case를 분석하다 경준님과 의견이 나온 사안이 있어서 공유 드립니다!
주문을 처리할 때 주문자 정보를 필요로 하는데 비회원 주문을 허용할 시에는 비회원 주문자 정보를 관리하는 모듈이 또 필요할 것 같아요. 첫번째 스펙으로는 비회원 주문 없이 회원 주문만 받는 건 어떻게 생각하시나요?

**Comment by TedLeem** [comment]
회원 주문만 받는게 좋을 것 같네요!
